### DISCUSSION TITLE
Softcore achievement list

### DISCUSSION THREAD

**Discussion by coolscribble** [RetroAchievements/RAWeb#750] [Ideas]
Is it possible to make personal list of softcore achievements to see, what you need to won in hardcore?

**Comment by meleu** [comment]
In the "Completion Progress" box you can see the games with a progress bar, when the bar has a blue part it means you earned some cheevos only in softcore. Then you can click on the game and see which cheevos were earned in softcore only.

**Comment by coolscribble** [comment]
I know about that, but it would be handy in here "softcore" and "hardcore"
![obrazek](https://user-images.githubusercontent.com/43485409/47618092-3231a580-dacf-11e8-9503-c8fffa987132.png)

**Comment by Jamiras** [comment]
While it doesn't list the individual achievements that you've unlocked in softcore, you can go to "Completion Progress" and select the "Games with softcore awards" Status to only show games where you have softcore unlocks. Then you can visit those game pages to find the individual achievements.
